--- src/tdp-provider.c.orig	2010-09-14 12:26:54.842541554 +0200
+++ src/tdp-provider.c	2010-09-14 12:27:37.656514982 +0200
@@ -21,7 +21,7 @@
 // with this program.  If not, see <http://www.gnu.org/licenses/>.
 //##############################################################################
 #include <unistd.h>
-#include <thunar-vfs/thunar-vfs.h>
+#include <gio/gio.h>
 
 #include "tdp-provider.h"
 
@@ -160,9 +160,10 @@
 	GtkWidget * window,
 	GList * files)
 {
+	GFile * file;
 	GList * actions = NULL;
 	GList * lp;
-	gchar buffer[THUNAR_VFS_PATH_MAXSTRLEN];
+	gchar * path;
 
 	GList * filelist = NULL;
 
@@ -179,15 +180,14 @@
 
 	for(lp = files; lp != NULL; lp = lp->next)
 	{
-		ThunarVfsInfo * info = thunarx_file_info_get_vfs_info(lp->data);
-		thunar_vfs_path_to_string(info->path, buffer,
-			THUNAR_VFS_PATH_MAXSTRLEN, NULL);
-		thunar_vfs_info_unref(info);
+		file = thunarx_file_info_get_location(lp->data);
+		path = g_file_get_path (file);
+		g_object_unref (file);
 
 		dropbox_write(io_channel, "\t");
-		dropbox_write(io_channel, buffer);
+		dropbox_write(io_channel, path);
 
-		filelist = g_list_append(filelist, g_strdup(buffer));
+		filelist = g_list_append(filelist, path);
 	}
 
 	dropbox_write(io_channel, "\ndone\n");
