--- wscript.orig	2010-09-14 12:26:29.556764086 +0200
+++ wscript	2010-09-14 12:27:54.347541040 +0200
@@ -12,8 +12,8 @@
 
 def configure(conf):
 	conf.check_tool('compiler_cc')
-	conf.check_cfg(package='thunarx-1', uselib_store='THUNARX', mandatory=True, args='--cflags --libs')
-	conf.check_cfg(package='thunar-vfs-1', uselib_store='THUNARVFS', mandatory=True, args='--cflags --libs')
+	conf.check_cfg(package='thunarx-2', uselib_store='THUNARX', mandatory=True, args='--cflags --libs')
+	conf.check_cfg(package='gio-2.0', uselib_store='GIO', mandatory=True, args='--cflags --libs')
 
 def build(bld):
 	prog = bld.new_task_gen('cc', 'cshlib')
@@ -23,4 +23,4 @@
 	prog.includes = 'src'
 	prog.find_sources_in_dirs('src')
 	bld.install_files('${PREFIX}/share/icons/hicolor/16x16/apps', 'data/icons/hicolor/16x16/apps/thunar-dropbox.png')
-	bld.install_as('/usr/lib/thunarx-1/thunar-dropbox.so', 'libthunar-dropbox.so')
+	bld.install_as('${PREFIX}/lib/thunarx-2/thunar-dropbox.so', 'libthunar-dropbox.so')
