From 94b3a3c6a0cfdcdbd98edce055d5c83ecb361f37 Mon Sep 17 00:00:00 2001
From: acxz <17132214+acxz@users.noreply.github.com>
Date: Thu, 4 Aug 2022 20:10:53 -0400
Subject: [PATCH] ifndef `wxID_OSX_*` for erlang 25.0

---
 src/wings_frame.erl | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/wings_frame.erl b/src/wings_frame.erl
index f6c13c71..3ec6bb83 100644
--- a/src/wings_frame.erl
+++ b/src/wings_frame.erl
@@ -37,10 +37,18 @@
 -define(IS_SPLIT(WinProp), (element(1, WinProp) =:= split
 			    orelse element(1, WinProp) =:= split_rev)).
 
+-ifndef(wxID_OSX_HIDE).
 -define(wxID_OSX_HIDE, 5250).
+-endif.
+-ifndef(wxID_OSX_HIDEOTHERS).
 -define(wxID_OSX_HIDEOTHERS, 5251).
+-endif.
+-ifndef(wxID_OSX_SHOWALL).
 -define(wxID_OSX_SHOWALL, 5252).
+-endif.
+-ifndef(wxID_OSX_MENU_LAST).
 -define(wxID_OSX_MENU_LAST, 5255).
+-endif.
 
 %% API  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-- 
2.39.2

